.github/workflows/main.yml

```diff
@@ -43,7 +41,7 @@ jobs:
               --verbose \
               --runtime=host \
               --cov=pyodide_build --cov=pyodide \
-              src pyodide-build packages/micropip/ packages/_tests tools/
```

Comment:
Is there a reason you deleted `packages/micropip` and `packages/_tests` here?

Reply:
I put back `packages/_tests`. `packages/micropip` is not needed as there is no host test in micropip anyway.